strapi/src/components/blocks/banner-basic-button.json

```diff
@@ -0,0 +1,16 @@
+{
+  "collectionName": "components_blocks_banner_basic_buttons",
```

Comment:
Please create and use common `basic link` component in Blocks that contains `label` (required), `page` relation and `url`.